windows/renderer/styles.css

```
@@ -0,0 +1,7439 @@
+  opacity: .72;
+}
+
+.pane-grip::before {
+  content: "";
+  position: absolute;
+  left: 3px;
+  top: 2px;
+  width: 2px;
+  height: 2px;
+  border-radius: 999px;
+  background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)